crates/epaint/src/tessellator.rs

```diff
@@ -864,19 +875,58 @@ fn stroke_path(
     feathering: f32,
     path: &[PathPoint],
     path_type: PathType,
-    stroke: Stroke,
+    stroke: &PathStroke,
     out: &mut Mesh,
 ) {
     let n = path.len() as u32;
 
-    if stroke.width <= 0.0 || stroke.color == Color32::TRANSPARENT || n < 2 {
+    if stroke.width <= 0.0 || stroke.color == ColorMode::Solid(Color32::TRANSPARENT) || n < 2 {
         return;
     }
 
     let idx = out.vertices.len() as u32;
 
+    let min_x = path
+        .iter()
+        .map(|point| OrderedFloat::from(point.pos.x))
+        .min()
+        .map(OrderedFloat::into_inner)
+        .unwrap();
+    let max_x = path
+        .iter()
+        .map(|point| OrderedFloat::from(point.pos.x))
+        .max()
+        .map(OrderedFloat::into_inner)
+        .unwrap();
+    let min_y = path
+        .iter()
+        .map(|point| OrderedFloat::from(point.pos.y))
+        .min()
+        .map(OrderedFloat::into_inner)
+        .unwrap();
+    let max_y = path
+        .iter()
+        .map(|point| OrderedFloat::from(point.pos.y))
+        .max()
+        .map(OrderedFloat::into_inner)
+        .unwrap();
```

Comment:
```suggestion
    let bbox = Rect::from_poins(path);
```

Reply:
I was considering putting the bounding box in a once_cell::Lazy to potentially save on having to do the bounding box calculations if it's not needed, do you think that's necessary?

Reply:
the simplify suggestion kind of renders this moot, i'll investigate with the new benchmarks later.

(66644820a93a76758da3d8f8f5d7b03ba88f79a9)